package.json

```diff
@@ -47,6 +47,10 @@
     "create-react-class": "15.5.3",
     "creditcards": "2.1.2",
     "cross-env": "4.0.0",
+    "d3-array": "1.2.0",
```

Comment:
@blowery I ran `make shrinkwrap` but it seems to have touched a bunch of other packages (see above). Read up a bit here - https://github.com/Automattic/wp-calypso/blob/a70587157b33783ba4d5e0e4a1e136680d459d51/docs/shrinkwrap.md - not sure if there is a way to isolate to specific packages?